native/src/components/CitySelector.tsx

```diff
@@ -26,6 +26,12 @@ const SearchBar = styled.View`
   padding: 0 10%;
 `
 
+const SearchCounter = styled.Text`
+  margin: 15px 0 10px 0;
```

Comment:
We have a linting rule for shorthand properties, please use either of the following:
```suggestion
  margin: 15px 0 10px;
```
or
``` suggestion
  margin-top: 15px;
  margin-bottom: 10px;
```
Just a side note: We generally try to use sizes/paddings that are a factor of 2, i.e. 2, 4, 8, 16, 32, ... or their composites (6, 12, 24, ...) if possible